backend/src/domain/Games.js

```diff
@@ -0,0 +1,58 @@
+import {Game} from './models/Game';
+
+let games = null;
```

Comment:
not sure if its best practice to have a function declared by itself, when a class is being used. (I know its not exaclty linked, but still). Also, is the let games = null, storing all the games (including ones that have finished?)

Reply:
Yea, not sure if its good practise either.
I think its only storing the active games.

Reply:
why are we storing games as a variable outside the class scope, if we already have activeGames in the class scope?

Reply:
we could use a cached singleton, as stated here i think https://medium.com/swlh/node-js-and-singleton-pattern-7b08d11c726a

Reply:
> why are we storing games as a variable outside the class scope, if we already have activeGames in the class scope?

Games class has mutliple game objects. We still need to store an instance of the Games class somewhere.

Reply:
> we could use a cached singleton, as stated here i think https://medium.com/swlh/node-js-and-singleton-pattern-7b08d11c726a.
> 
> thoughts?

I ran out of free stories for medium D:

Reply:
oh i understand now. But yea, if we just export the the games class, such as module.exports = new Games(); it should work as a singleton.

Reply:
however, if that doesn't work, we change just have two classes, one that is "private" by not exporting it, and one that is "public" by exporting it

Reply:
will module.export = new Games() always return the same Games instance?

Reply:
okay changed it to use export default new Games()